### DISCUSSION TITLE
How to access window/navigator object of the snap dialog box

### DISCUSSION THREAD

**Discussion by Arch0125** [MetaMask/snaps#1764] [Q&A]
I'm currently building a passkeys snap and for that i would require to access those objects, tried accessing them through index.ts but since it is a server, it won't be having them.
Is there any bypass to access window/navigator from the snap dialog context

**Comment by Montoya** [comment]
Sorry, that's not possible. Snaps run in a sandbox and have no access to browser globals like window or navigator. The context does not exist for a Snap. We would have to provide an API for a Snap to have access to that.
